python/pyspark/sql/dataframe.py

```diff
@@ -1181,7 +1181,7 @@ def approxQuantile(self, col, probabilities, relativeError):
         Space-efficient Online Computation of Quantile Summaries]]
         by Greenwald and Khanna.
 
-        :param col: the name of the numerical column
+        :param cols: the name(s) of the numerical column(s)
```

Comment:
Not sure offhand on what the general consensus is on documenting multiple potential input types. But perhaps we can specify that cols can be a single column (str) or a list of col names.